### PR TITLE
Temporary fix to get Gallery images to fit Nibble

### PR DESCRIPTION
**Pull request by Torrib**
Kanskje min mest sexy pull request.

### CODE REVIEW

File: apps/gallery/util.py [hunk 1]
```diff
@@ -153,6 +156,9 @@ def create_thumbnail(source_image_path, destination_thumbnail_path, size):
 
 def resize_image(source_image_path, destination_thumbnail_path, size):
     status = _open_image_file(source_image_path)
+
+    print status
```
Comment (OptimusCrime) [inline apps/gallery/util.py]: sweet debug!

**Comment by OptimusCrime** [comment]
Does this in any way fuck up images elsewhere?

**Comment by Torrib** [comment]
It wont change the existing images but it will prevent new images in 16/9 from being uploaded.
This is only a temporary fix while we add the images needed for the inventory then it should be reverted back while we work on a more permanent solution.

File: apps/gallery/util.py [hunk 2]
```diff
@@ -248,7 +252,7 @@ def crop_image(source_image_path, destination_path, crop_data):
 
     # Check all the dimension and size things \o/
     # Give an epsilon of 0.01 because calculations.
-    if crop_width / crop_height < float(16)/float(9) - 0.01 or crop_width / crop_height > float(16)/float(9) + 0.01:
+    if crop_width / crop_height < float(ratio_x)/float(ratio_y) - 0.01 or crop_width / crop_height > float(ratio_x)/float(ratio_y) + 0.01:
```
Comment (hernil) [inline apps/gallery/util.py]: Line length

**Comment by OptimusCrime** [comment]
Ah, damn son, that is indeed dirty

**Comment by sklirg** [comment]
Let's wait for Frigg build status.

**Comment by hernil** [comment]
So yeah, I just got cancer. It's temporarily deployed. Will revert asap.

**Comment by myth** [comment]
So yea, how bad was it? Do we have to call an ambulance?

**Comment by hernil** [comment]
It's done, and we do not want this in develop.

**Pull request closed without merging**